### DISCUSSION TITLE
搭建新官网 转移Wiki平台

### DISCUSSION THREAD

**Discussion by Fgaoxing** [Qexo/Qexo#92] [Show and tell]
# #  _**内容被删！!**_

**Comment by am-abudu** [comment]
有考虑过 但没有相应计划 目前依托于Github提供的相关服务

**Reply by Fgaoxing** [reply]
我已经在做了！！

**Reply by Fgaoxing** [reply]
我还没发到GitHub上

**Reply by am-abudu** [reply]
没有这个打算 感谢你的好意 但我不会对此给予支持

**Reply by Fgaoxing** [reply]
那个把Wiki的Markdown版发到[admin@ty-blogt.top](mailto:admin@ty-blogt.top)

**Reply by Fgaoxing** [reply]
我已经写完了，就差wiki了！

**Reply by am-abudu** [reply]
> 那个把Wiki的Markdown版发到[admin@ty-blogt.top](mailto:admin@ty-blogt.top)

Github的Wiki是开源的 你可以直接在页面上找到Clone

**Reply by Fgaoxing** [reply]
加了英语文档
展示：
![image](https://user-images.githubusercontent.com/92655031/154040160-f26d30fd-1141-45e0-bf09-3d1ff814ba93.png)

**Reply by Fgaoxing** [reply]
你只要部署到你的域名上即可！！
英文文档可以使用，电子科技翻译https://fanyi.baidu.com/ ,需切换成电子科技模式!

**Reply by Fgaoxing** [reply]
请通过该提议，谢谢

**Reply by am-abudu** [reply]
但是Qexo目前还没有对多语言支持做好准备 并且我并不是很喜欢翻译器给出的翻译
这将是一个非官方项目 但我很喜欢他 :)
